src/lang/en-US.ts

```diff
@@ -1078,9 +1078,35 @@ const SchemaEditor_Collection: ResolvedIntlConfig['messages'] = {
 const EntityEvolution: ResolvedIntlConfig['messages'] = {
     'entityEvolution.failure.errorTitle': `Update Failed`,
     'entityEvolution.serverUnreachable': `${CommonMessages['common.failedFetch']} while trying to update collections`,
-    'entityEvolution.error.title': `Changes Rejected Due to Incompatible Schema Updates`,
-    'entityEvolution.error.message': `Schema changes will break downstream tasks. To avoid this, click below and then publish a new version of the affected collections.`,
-    'entityEvolution.error.note': `Note: This may result in additional cost as new collection versions are backfilled.`,
+    'entityEvolution.error.title': `Changes rejected due to incompatible collection updates`,
+    'entityEvolution.error.message': `The proposed collection changes would break downstream tasks. You can click '${CTAs['cta.evolve']}' below to automatically update your draft with the following recommended actions.`,
+    'entityEvolution.error.note': `Note: This may result in additional cost as new versions are backfilled.`,
+
+    // Single quotes are special and must be doubled: https://formatjs.io/docs/core-concepts/icu-syntax#quoting--escaping
+    'entityEvolution.action.recreateOneBinding.description': `the Materialization ''{materializationName}'' will be updated to materialize the collection into a new resource`,
+    'entityEvolution.action.recreateBindings.description': `{materializationCount} {materializationCount, plural,
+        one {Materialization}
+        other {Materializations}
+    } will be updated to materialize the collection into new resources`,
```

Comment:
@psFried  - added support for plurals vs singular here. I know this is actually handled within the component but just trying to always make this a good habit so if anything changes later the messages won't get messed up.

Thee library has a lot of options but the main 3 we're trying to handle  are: `zero` (sometimes), `one`, and `other`. 
https://formatjs.io/docs/core-concepts/icu-syntax/#plural-format

Reply:
SGTM, thanks. And makes sense that we'd want to support plurals universally here, so we never have to wonder whether a message does so or not.